FizzBuzz.js

```diff
@@ -0,0 +1,26 @@
+// FizzBuzz
+
+const FizzBuzz = () => {
+    let string;
+    for(let i = 1; i <= 100; i++) {
+        string = '';
+
+        if(i % 3 == 0)
+            {
+                string += 'Fizz';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306921)
```suggestion
                string += "Fizz";
```